tests/cli/commands/test_user_command.py

```diff
@@ -45,18 +48,9 @@ def _set_attrs(self, app, dagbag, parser):
         self.dagbag = dagbag
         self.parser = parser
         self.appbuilder = self.app.appbuilder
-        self.clear_roles_and_roles()
+        delete_users(app)
```

Comment:
Is this change going to have any side affects on other tests? The two functions do pretty different things.

`clear_roles_and_roles` (maybe this should have been names `clear_users_and_roles`?) only deletes the test users and roles used in these tests

`delete_users` deletes all users (and leaves the roles intact).

Reply:
Int this test suite no new/test roles are created and hence no need to delete any roles. 
Besides, `clear_roles_and_roles` function is used inside `test_role_command.py` suite where it only deletes specific users and roles and when new users are added or imported here they were not deleted and hence this PR cleans those traces.